case-lib/lib.sh

```diff
@@ -768,4 +768,14 @@ set_alsa_settings()
     esac
 }
 
+reset_PGA_volume()
+{
+    # set all PGA* volume to 0dB
+    amixer -Dhw:0 scontrols | sed -e "s/^.*'\(.*\)'.*/\1/" |grep PGA |
```

Comment:
for IPC4 the volume control are named as gain, I think you should also grep gain here.

Reply:
Yes, you are right. But the volume-base-test.sh only support PGA control ATM. The volume test is skipped on all our IPC4 platforms. I can submit another PR for IPC4.